core/unit_test/TestCommonPolicyConstructors.hpp

```diff
@@ -87,7 +84,7 @@ void test_prefer_desired_occupancy() {
       !decltype(policy_drop_occ)::experimental_contains_desired_occupancy);
 }
 
-TEST(TEST_CATEGORY, execution_policy_occupancy_and_hint) {
+[[maybe_unused]] void test_execution_policy_occupancy_and_hint() {
```

Comment:
Why did you change this and removed the assertions above?

Reply:
So this can all be compile-time checks. I moved the runtime checks to the new test.

Reply:
Undo all changes to that file